.github/workflows/build.yml

```diff
@@ -26,9 +26,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/setup-python@v2
         with:
           fetch-depth: 0  # required for versioneer to find tags
```

Comment:
We could probably remove this as well as we don't need versioneer to find tags here no ?

Reply:
Yes, that might be better. What about skip_existing on the upload? We don't need that anymore?

Reply:
> What about skip_existing on the upload? We don't need that anymore?

Since we can retrieve the built artifacts and upload the remaining wheels in case of error during upload, I think we can drop this safely.

Reply:
It was also originally there because Travis CI could upload first.